.gitignore

```diff
@@ -1 +1,3 @@
 /bin/
+*.swp
+*dev.log
```

Comment:
I don't think the installer repo needs to add user-specific files to git ignore.

Reply:
I agree about the dev.log, will drop it
Regarding the *.swp, this is temporary file used by the vi/vim editor and might be accidentally added to commits. as this type file never should be added, isn't it good enough reason to add it (and maybe also other editor's specific files)

Reply:
I think people should use https://stackoverflow.com/a/1753078 to add any editor ignore files. Forcing repos to include ignore files for all editors is too much of a slippery slope imo, so i would personally not add anything here.

Reply:
Thanks @abhinavdahiya 
I removed this change